src/App.js

```diff
@@ -1,21 +1,15 @@
-import React from 'react';
-
+import EntryList from "./EntryList";
 import './App.css';
-function MyList(props) {
-  const items = props.items;
-  const listItems = numbers.map((item) =>
-    <li>{item}</li>
-  );
-  function App() {
-    return (
-      <ul>{listItems}</ul>
-    );
-  }
-  const items = [item1, item2, item3, item4, item5];
-  ReactDOM.render(
-    <MyList items={items} />,
-    document.getElementById('root')
-  );
+const fruits = ["carrot"]
```

Comment:
this is good, you have created fruits array 
this array can be passed to `EntryList`

Reply:
Okay